index.html

```diff
@@ -136,21 +136,21 @@ <h2>About me</h2>
     </div>
     <ul class="knowledge">
       <ul>
-        <li><h2><img src="./Normal_Button/language-rectangle.png"> Language</h2></li>
+        <li><h3><img src="./Normal_Button/language-rectangle.png"> Language</h3></li>
         <li>Javascript</li>
         <li>Ruby</li>
         <li>HTML</li>
         <li>CSS</li>
       </ul>
       <ul>
-        <li><h2><img src="Normal_Button/frameworks-rectangle.png">   Frameworks</h2></li>
+        <li><h3><img src="Normal_Button/frameworks-rectangle.png">   Frameworks</h3></li>
         <li>React.js</li>
         <li>Ruby on Rails</li>
         <li>RSpec</li>
         <li>Capybara</li>
         </ul>
       <ul>
-        <li><h2><img src="Normal_Button/skills-circle.png"> Skills</h2></li>
+        <li><h3><img src="Normal_Button/skills-circle.png"> Skills</h3></li>
```

Comment:
Kindly consider adding the `alt` attribute. The `alt` attribute specifies an alternate text for an image, if the image cannot be displayed.
The `alt` attribute provides alternative information for an image if a user for some reason cannot view it (because of a slow connection, an error in the `src` attribute, or if the user uses a screen reader).

Reply:
Hi @Strangeal ,
Thank you for your review! I do have one concern
![image](https://user-images.githubusercontent.com/117373513/230233684-a40dbf1b-8060-417a-a6a9-5dadbd30fcbe.png)
As you can see in this image, the img elements that you have mentioned are purely decorative. From what I understand, these types of images are not required to have alt text. If they do require alt text please let me know so I can fix them with "Orange shape" alt text.